### ISSUE TITLE
Can't install openjdk@ on macOS

### ISSUE THREAD

**Issue by tmurakam** [shyiko/jabba#187]
"jabba install openjdk@1.11.0-11" fails on macOS.
It also fails for 1.10.0 and 1.10.0-1.

The all openjdk tgz for macOS contains some paths other than "jdk-xxx.jdk/Contents/Home" (for example, "jdk-xxx.jdk/Contents/Info.plist"), so I think stripping of the directory is failed.

**Comment by shyiko** [comment]
Thank you for letting me know, Takuya. Fixed in 0.9.3.

**Comment by charypar** [comment]
0.9.3 seems to have broken the installation for oracle JDK on macOS:

```
jabba install 1.8
Downloading 1.8.172 (http://download.oracle.com/otn-pub/java/jdk/8u172-b11/a58eab1ec242421181065cdc37240b08/jdk-8u172-macosx-x64.dmg) 
259908002/259908002
Mounting /var/folders/yy/2p71gc4171731cwj7lk9mbwh0009v6/T/jabba-d-553085442 
Extracting /var/folders/yy/2p71gc4171731cwj7lk9mbwh0009v6/T/jabba-d-553085442 to /Users/43980819/.jabba/jdk/1.8.172 
Unmounting /var/folders/yy/2p71gc4171731cwj7lk9mbwh0009v6/T/jabba-d-553085442 
stat /Users/43980819/.jabba/jdk/1.8.172/Home/bin/java: no such file or directory 
```

It seems to be checking for the java binary in the wrong path (missing Contents before Home)

**Comment by shyiko** [comment]
@charypar sorry about that. Fixed in 0.9.4.
